src/audio/mixer.c

```diff
@@ -408,7 +476,7 @@ static int mixer_prepare_common(struct comp_dev *dev)
 #endif /* CONFIG_FORMAT_S16LE */
 #if CONFIG_FORMAT_S24LE
 		case SOF_IPC_FRAME_S24_4LE:
-			md->mix_func = mix_n_s32;
+			md->mix_func = mix_n_s24;
```

Comment:
This could have been a separate commit (and even PR) as well.

Reply:
@dbaluta shared your thoughts. If preferred I can do one more iteration The system/merge/build step still fails so this can't be merged yet. Please respond if separate PR or commit desired.

Reply:
can we split this for bisect. Thanks.

Reply:
I'll go for 1.